CHANGELOG.md

```diff
@@ -3,7 +3,153 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-## v7.0.0-beta.3
+## 7.0.0-beta.4
+
+_Feb 23, 2024_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Introduce a new DOM structure for the field components that provides a better accessibility
+- 🚀 Simplify Data Grid DOM structure for improved performance (#12013) @romgrk
+- 🐞 Bugfixes
+- 📚 Documentation improvements
+
```

Comment:
May be add it to the top level breaking changes:
```suggestion

### Breaking changes

- Legacy IE11 bundle has been removed for all MUI X components

```

Reply:
Do we want to suggest some workaround?
CC @flaviendelangle @LukasTy

Reply:
Given the descriptive highlight, I feel this doesn't make sense anymore.

Reply:
Maybe not `components`, but `packages`? 🤔

Reply:
Ended up adding it, we could always follow-up the conversation and update the changes.
I'm gonna release it now.